### PR TITLE
Added exclude reserved files option

### LINKED ISSUE #5: Add option to show only files that aren't reserved

### PR DESCRIPTION
**Pull request by Seluj78**
Closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ optional arguments:
   -f, --fuzzy           print only files marked as fuzzys
   -o, --offline         don't perform any fetching to GitHub/online
   -n, --no-reserved     don't print info about reserved files
-  -c, --counts          render list with the count of remaining entries
-                        (translate or review) rather than percentage done
+  -c, --counts          render list with the count of remaining entries (translate or review) rather than percentage done
   -j, --json            format output as JSON.
+  --not-reserved        Will print out only files that arent reserved
```
Comment (christopheNan) [inline README.md:32]:
```suggestion
  --not-reserved        select only files that aren't reserved
```

Comment (christopheNan) [inline README.md:32]:
```suggestion
  --not-reserved        select only files that aren't reserved
```

File: potodo/potodo.py [hunk 2]
```diff
@@ -314,6 +323,16 @@ def main() -> None:
...
     )
 
+    parser.add_argument(
+        # "-x"
+        # Should we add a quick one dash option for not reserved ?
+        #  As it will be used a lot
+        "--exclude-reserved",
+        action="store_true",
+        dest="exclude_reserved",
+        help="Will print out only files that arent reserved",
```
Comment (christopheNan) [inline potodo/potodo.py:333]:
```suggestion
        help="select only files that aren't reserved",
```

**Review by christopheNan — APPROVED** [review]
👍 ❤️

**Comment by Seluj78** [comment]
Need another review @christopheNan

**Review by christopheNan — APPROVED** [review]

File: potodo/potodo.py [hunk 3]
```diff
@@ -332,6 +338,16 @@ def main() -> None:
         help="Will exclude files with fuzzies from output.",
     )
 
+    parser.add_argument(
+        # "-x"
+        # Should we add a quick one dash option for not reserved ?
+        #  As it will be used a lot
```
Comment (Seluj78) [inline potodo/potodo.py:344]:
```suggestion
```

**Pull request merged**